tests/integration/test_replay_events.py

```diff
@@ -136,3 +136,22 @@ def test_replay_events_without_processing(mini_sentry, relay_chain):
 
     replay_event = envelope.items[0]
     assert replay_event.type == "replay_event"
+
+
+def test_replay_event_with_processing_other(
```

Comment:
Why `_other`? Wdyt about a test name to reflect the expectation of filtering out the replay?

Reply:
I copied the test above and renamed it so they wouldn't conflict.  And then forgot to actually name it.  My bad!